docs/core/tools/global-json.md

```diff
@@ -1,32 +1,96 @@
 
-For example:
+Note that this field:
+
+- Doesn't have globbing support, that is, the full version number has to be specified.
+- Doesn't support version ranges.
+
+The following example shows the contents of a *global.json* file:
 
 ```json
 {
   "sdk": {
-    "version": "1.0.0-preview2-003121"
+    "version": "2.1.300"
   }
 }
 ```
+
+## global.json and the .NET Core CLI
+
+It's helpful to know which versions are available in order to set one in the *global.json* file. You can find the full list of supported available SDKs at the [.NET Downloads](https://www.microsoft.com/net/download/all) site. Starting with .NET Core SDK 2.1, you can run the following command to verify which SDK versions are already installed on your machine:
+
+```console
+dotnet --list-sdks
+```
+
+To install additional .NET Core SDK versions on your machine, visit the [.NET Downloads](https://www.microsoft.com/net/download/all) site.
+
+You can create a new the *global.json* file in the current directory by executing the [dotnet new](dotnet-new.md) command, similar to the following example:
+
+```console
+dotnet new globaljson --sdk-version 2.1.300
+```
+
+## Matching rules
+
+Starting with .NET Core SDK 2.0, the following rules apply when determining which version of the SDK to use:
+
+- If no *global.json* file is found or *global.json* doesn't specify an SDK version, the latest installed SDK version is used. Latest SDK version can be either release or pre-release - the highest version number wins.
+- If *global.json* does specify an SDK version:
+  - If the specified SDK version is found on the machine, that exact version is used.
+  - If the specified SDK version can't be found on the machine, the latest installed SDK **patch-version** of that version is used. Latest installed SDK **patch-version** can be either release or pre-release - the highest version number wins.
+  - If the specified SDK version and an appropriate SDK **patch-version** can't be found, an error is thrown.
+
+The SDK version is currently composed of the following parts:
+
+`[major].[minor].[xyz][-optional preview name]`
+
+The **patch-version** is defined by the last two digits (`yz`) in the last portion of the number (`xyz`) for SDK versions 2.1.100 and greater. For example, if you specify `2.1.300` as the SDK version, it can try to find up to `2.1.399` but `2.1.400` isn't considered a patch version for `2.1.300`.
+
+.NET Core SDK 2.0 presented a slightly different behavior. If the specified version wasn't present, the highest patch version was selected, whether it was higher or lower than the one requested. Also, .NET Core SDK 2.0 versions 2.1.100 through 2.1.201 were feature versions, but treated as patch versions for the *global.json* SDK selection. Both of these unusual behaviors have been fixed for later versions. Also, for SDK versions earlier than 2.1.100, the **patch-version** was defined by the third position (`z`) in the last portion of the number (`xyz`).
```

Comment:
I think this is my text, sorry, but it comes out complex. The text above is actually the 2.0 behavior and we can clarify just by adding a simple sentence on 2.1. People expect we stated the 2.1 behavior (they don't expect lower patches tone used. 

So, with that change, let's drop this paragraph and add this:

.NET Core SDK 2.1.100 through 2.1.201 were released during the transition between version number schemes and do not correctly handle the `xyz` notation. We highly recommend if you specify these versions in global.json that you ensure the specified versions is on target machines.